uk.co.compucorp.civicrm.hrleaveandabsences/views/shared/components/leave-balance-tab-filters.html

```diff
@@ -1,34 +1,20 @@
-<div class="chr_leave-balance-tab__filters panel panel-default">
-  <header class="panel-heading panel-heading-plain">
-    <h3 class="panel-title">
-      <i class="fa fa-chevron-down" aria-hidden="true"></i>
-      Show Filters
-    </h3>
-  </header>
-  <div class="panel-collapse">
-    <form class="panel-body form-inline">
-      <div class="row">
-        <div class="form-group col-sm-4">
-          <label>Periods</label>
-          <select class="form-control no-select2"
-            ng-model="balanceFilters.filters.absence_period"
-            ng-options="period.id as period.title for period in balanceFilters.absencePeriods | orderBy:'title'">
-          </select>
-        </div>
-        <div class="form-group col-sm-4">
-          <label>Leave Type</label>
-          <select class="form-control no-select2"
-            ng-model="balanceFilters.filters.absence_type"
-            ng-options="type.id as type.title for type in balanceFilters.absenceTypes | orderBy:'title'">
-          </select>
-        </div>
-      </div>
-      <div class="text-right">
-        <hr />
-        <button type="submit" class="btn btn-primary" ng-click="balanceFilters.submitFilters()">
-          Filter
-        </button>
-      </div>
-    </form>
+<div class="col-md-3 chr_leave-balance-tab__filter">
+  <div class="crm_custom-select crm_custom-select--full">
+    <select class="form-control no-select2"
+      ng-change="balanceFilters.submitFilters()"
+      ng-model="balanceFilters.filters.absence_type"
+      ng-options="type.id as type.title for type in balanceFilters.absenceTypes | orderBy:'title'">
+    </select>
+    <span class="crm_custom-select__arrow"></span>
+  </div>
+</div>
+<div class="col-md-3 col-md-offset-6 chr_leave-balance-tab__filter">
```

Comment:
the `col-md-offset-6` should be removed. These filters need to be one next to the other.

Reply:
As per discussion with @jamienovick, we decided to place them exactly like that.